extensions/arc/src/models/miaaModel.ts

```diff
@@ -100,25 +99,14 @@ export class MiaaModel extends ResourceModel {
 		try {
 			try {
 				let result;
-				if (this.controllerModel.info.connectionMode.toLowerCase() === ConnectionMode.direct) {
-					result = await this._azApi.az.sql.miarc.show(
-						this.info.name,
-						{
-							resourceGroup: this.controllerModel.info.resourceGroup,
-							namespace: undefined
-						},
-						this.controllerModel.azAdditionalEnvVars
-					);
-				} else {
-					result = await this._azApi.az.sql.miarc.show(
-						this.info.name,
-						{
-							resourceGroup: undefined,
-							namespace: this.controllerModel.info.namespace
-						},
-						this.controllerModel.azAdditionalEnvVars
-					);
-				}
+				result = await this._azApi.az.sql.miarc.show(
+					this.info.name,
+					{
+						resourceGroup: undefined,
```

Comment:
If these aren't used then I would suggest making them optional properties (if they aren't already) and just leaving them out of these definitions.